users/views.py

```diff
@@ -202,3 +208,17 @@ def update_organization_user(sender, instance: OrganizationUser, **kwargs):
     async_to_sync(channel_layer.group_send)(
         user_group_name, {"type": "send_user", "data": user_data}
     )
+
+
+@api_view(["POST"])
+def retrieve_analytics_app_access_token(request):
+    """ Makes a client credentials request to Auth0 app to get an access token. """
+    url = AUTH0_TOKEN_URL
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": AUTH0_CLIENT_ID,
+        "client_secret": AUTH0_CLIENT_SECRET,
+        "audience": AUTH0_AUDIENCE,
+    }
+    response = requests.post(url, data=payload)
+    return Response(response.json(), status=status.HTTP_200_OK)
```

Comment:
is the JWT being generated by Auth0 as well here?

Reply:
@dalmia yes, the JWT is being generated by Auth0. The response will be something like this:
```json
{
 "access_token": "token",
 "type": "Bearer",
 "expiry": 86400,
}
```